Example/ViewController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  ViewController.swift
+//  Example
+//
+//  Created by David Hardiman on 13/02/2016.
+//  Copyright © 2016 David Hardiman. All rights reserved.
+//
+
+import UIKit
+import Fetch
+
+class ViewController: UIViewController {
```

Comment:
This demonstrates how to use it. No magic, just create a model object that knows how to parse the response and pass the type to `get`